From d153b3b09615d850dbced3ff6362b3117182821d Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Sat, 25 Jan 2014 18:19:32 +0100
Subject: [PATCH 47/69] DOC: fix misleading information about SIGQUIT

SIGQUIT dumps the pools state to stderr, not to the logs. Thanks to
Jim Freeman for reporting this.
---
 doc/haproxy.1 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/haproxy.1 b/doc/haproxy.1
index c9f159a..ee29c03 100644
--- a/doc/haproxy.1
+++ b/doc/haproxy.1
@@ -181,7 +181,7 @@ Both signals can be used to quickly stop the daemon.
 Dumps the status of all proxies and servers into the logs. Mostly used for trouble-shooting purposes.
 .TP
 \- \fBSIGQUIT\fP
-Dumps information about memory pools into the logs. Mostly used for debugging purposes.
+Dumps information about memory pools on stderr. Mostly used for debugging purposes.
 .TP
 \- \fBSIGPIPE\fP
 This signal is intercepted and ignored on systems without \fBMSG_NOSIGNAL\fP.
-- 
1.8.3.2

